### ISSUE TITLE
Mangago no scheme was found

### ISSUE THREAD

**Issue by PlaywithKen** [keiyoushi/tachiyomi-extensions-source#4005]
### Source information

Mangago 1.4.15

### Source language

English

### Steps to reproduce

When opening a story, there are onky the first page but throughout the whole chapter are all image could not load.
![Screenshot_20240711_103435_Mihon](https://github.com/user-attachments/assets/4e1099d0-d711-4e39-b83f-495bb19a62e4)

### Expected behavior

All chapters should be loaded.

### Actual behavior

The chapters is not loading and can only read in webview.

### Mihon/Tachiyomi version

0.16.5

### Android version

13

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open or closed issue.
- [X] I have written a short but informative title.
- [X] I have updated the app to version **[0.15.3](https://github.com/mihonapp/mihon/releases/latest)**.
- [X] I have updated all installed extensions.
- [X] I have tried the [troubleshooting guide](https://tachiyomi.org/help/guides/troubleshooting/).
- [X] If this is an issue with the app itself, I should be opening an issue in the [app repository](https://github.com/tachiyomiorg/tachiyomi/issues/new/choose).
- [X] I will fill out all of the requested information in this form.

**Comment by AwkwardPeak7** [comment]
Try logging in to your account in webview, see if that improves it.

**Comment by NLAIV** [comment]
Getting this same or similar issue, and might have narrowed its cause down.

Chapters with many images ("pages")  on separate pages (on webview) are affected. For example, half of the chapter's images are on page 1, and the rest are on second page. 

Many images on the first page seems to break the extension's ability to parse images from subsequent pages, causing the rest to fail to load.

Chapters with either all images on single page, or single image per page seem to be unaffected.


TLDR: many:many loads only 1st, rest will fail. 1:1 and ALL:1 are ok. (images:webpages)
